internal/services/network/express_route_port_resource.go

```diff
@@ -64,6 +57,11 @@ var expressRoutePortSchema = &pluginsdk.Schema{
 				Optional:     true,
 				ValidateFunc: validation.StringIsNotEmpty,
 			},
+			"macsec_sci_state": {
```

Comment:
state doesn't make much sense here? shoudl this be
```suggestion
			"macsec_sci_enabled": {
```